packages/opentelemetry-node/test/fixtures/use-grpc.proto

```diff
@@ -0,0 +1,42 @@
+// Copyright 2015 gRPC authors.
```

Comment:
why the license header is different from `./use-grpc.js`? both files are adapted from the same repo, right?

Reply:
Because this file is a direct copy. The other one is adapted (with perhaps significant-enough changes). IANAL, however. I added mention in the NOTICE file.
Tried to clarify in commit a2d671a and commit ddc6aad.